packages/calling-stateful-client/src/BetaToStableTypes.ts

```diff
@@ -41,4 +41,5 @@ export type TeamsCall = never | /* @conditional-compile-remove(teams-identity-su
  */
 export type IncomingCallCommon =
```

Comment:
If IncomingCallCommon is already stabilized, we can remove this class and always use that class from sdk